Automattic/wp-calypso
# fix: Disable copy site for sites that are not atomic

#### Proposed Changes

Don't allow copy site when a site has copy site feature but it's not atomic.

#### Testing Instructions

1. Create a site with a `Business` plan
2. Make sure that it's not atomic.
3. Create another site with a `Business` plan
4. Make sure that the site is atomic.

##### Test copy site from sites dashboard
1. Navigate to `/sites`
2.  Make sure that the newly created site (**not atomic**) _can't_ be copied (no menu item appears)
3. Make sure that the newly created site (**atomic**) _can_ be copied (menu item appears)

##### Test copy site from site tools
1. Navigate to Settings > General > Site Tools (**on your not atomic site**)
2. Make sure that "Copy Site" option _doesn't_ appear.
3. Navigate to Settings > General > Site Tools (**on your atomic site**)
4. Make sure that "Copy Site" option _appears_.




#### Pre-merge Checklist





fixes: https://github.com/Automattic/wp-calypso/issues/72707
